config/puma.rb

```diff
@@ -0,0 +1,15 @@
+workers Integer(ENV['WEB_CONCURRENCY'] || 2)
```

Comment:
in the first step, i would not use workers at all. you won't have that amount of traffic. Threads should do it and you simplify a lot of things